examples/static/app/css/app.css

```diff
@@ -0,0 +1,24 @@
+h1.title{
```

Comment:
Can you replace it with just: `.title` ? It has a better performance (small thing, but still :smile:  )